### PR TITLE
Add engineering-report template

### PR DESCRIPTION
**Pull request by manuelfuenmayor**
*From https://github.com/metanorma/mn-templates-ogc/issues/12*

For this template, I looked at several [engineering reports](https://www.ogc.org/docs/er#:~:text=Engineering%20Reports,or%20mandatory%20technology%20in%20procurements.) from OGC, and the structure varies quite a bit from each other. I took most common parts in my opinion.

Update:
Nick observations applied. @ronaldtse, if you don't have any further corrections, you can merge when you wish.

Thanks!

### CODE REVIEW

File: engineering-report/document.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
+= Engineering report for OGC (add title text)
+:comment: ### Document type; mandatory. Visit: https://www.metanorma.com/author/ogc/authoring/ for permitted types
+:doctype: engineering-report
+:comment: ### Language of the document; mandatory. Specified in two-letter code: "en" for English, "fr" for French
+:language: en
```
Comment (opoudjis) [inline engineering-report/document.adoc]: No evidence that OGC publishes outside of English. "en" is supplied by default; I would leave the attribute out, unless we know for a fact that OGC does publish outside of English.

File: engineering-report/document.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+:issued-date: 2018-07-10
+:comment: ### Date on which the standard was published; mandatory
+:published-date: 2018-06-12
+:comment: ### External link referencing the document; optional. If not provided, a default value is created following this structure: "http://www.opengis.net/doc/{abbrevation of doctype}/{abbrev}/{version}"
+:external-id: http://www.opengis.net/doc/PER/eoep-Hack2018
+:comment: ### Author one
+:fullname: John Doe
+:comment: ### Author two
+:fullname_2: Jane Doe
+:comment: ### Role of the authors; mandatory
```
Comment (opoudjis) [inline engineering-report/document.adoc]: Not mandatory; supplied only if the author is an editor. To make it clear that it applies to an individual author, make this :role_2:

**Review by opoudjis — COMMENTED** [review]
Looks pretty good to me, Manuel!

File: engineering-report/document.adoc (hunk 1 above)
Comment (manuelfuenmayor) [reply engineering-report/document.adoc]: Ok, done.

File: engineering-report/document.adoc (hunk 2 above)
Comment (manuelfuenmayor) [reply engineering-report/document.adoc]: Done, thanks.

**Pull request merged**